op-e2e/interop/supersystem.go

```diff
@@ -552,6 +594,44 @@ func (s *interopE2ESystem) prepareL2s() map[string]l2Set {
 	return l2s
 }
 
+// prepareContracts prepares contract-bindings for the L2s
+func (s *interopE2ESystem) prepareContracts() {
```

Comment:
I've been thinking of including the Emitter contract on chains by default and this makes me want to do it more